server/db/index.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const connectToDb = async () => {
+  mongoose.set("strictQuery", false);
+  mongoose.connect(
+    `mongodb+srv://${process.env.atlasUserName}:${process.env.atlasPassword}@khushi-blog-app.6h5resz.mongodb.net/?retryWrites=true&w=majority&appName=khushi-blog-app`,
```

Comment:
Ideally, entire connection string should be read from `.env`

Reply:
Sure